amundsen_application/static/js/components/Tags/TagsList/styles.scss

```diff
@@ -0,0 +1,35 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+@import 'variables';
+
+hr.header-hr {
+  border: 2px solid $brand-color-4;
+}
+
+.tags-list {
+  margin: 0 -4px;
+}
+
+@import 'variables';
+
+.browse-tags-link {
+  font-size: 16px;
+  text-decoration: none;
+  display: block;
+  margin-top: $spacer-1;
+
+  &:hover,
+  &:visited,
+  &:active,
+  &:link {
```

Comment:
Not important in this case, but the ordering is usually:
:link
:visited
:focus
:hover
:active 

LoVe Fears HAte

More on this here: https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Styling_links